### PR TITLE
Update package for 2025-2026

### PR DESCRIPTION
**Pull request by ericaporter**
Screenshot 2024-10-07 at 11 43 17

N.B. Do not merge - waiting for confirmation on amounts

### CODE REVIEW

File: lib/dfe/reference_data/itt/incentives.rb [hunk 1]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            bursary_amount: '30000',
+            scholarship: '32000',
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          'f0157f2e-6953-4e4b-b39b-575e34f1f975' => {
+            subject_name: 'physics',
+            itt_categories_id: 'd059a9cd-9b70-49bf-8194-9a71081e6cb9',
+            academic_cycle: ['2026'],
+            bursary_amount: '30000',
+            scholarship: '32000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:47]: update to
bursary_amount: '29000',
scholarship: '31000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 2]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          '0d497364-a854-40f5-be0d-bf490ab8cb1b' => {
+            subject_name: 'chemistry',
+            itt_categories_id: '167b889a-4cbb-47ff-a196-4103cb7ea43d',
+            academic_cycle: ['2026'],
+            bursary_amount: '28000',
+            scholarship: '30000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:55]: update to
bursary_amount: '29000',
scholarship: '31000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 3]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          '39134b18-dfd0-4ff5-b4d4-a50af6fd9e40' => {
+            subject_name: 'computing',
+            itt_categories_id: 'bc0a97ed-c09c-4688-9dce-b4ea888b6c51',
+            academic_cycle: ['2026'],
+            bursary_amount: '28000',
+            scholarship: '30000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:63]: update to
bursary_amount: '29000',
scholarship: '31000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 4]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          'ff42451b-a8a0-4078-b445-82dd249b7018' => {
+            subject_name: 'french',
+            itt_categories_id: '262f596b-b8a2-40c2-9efc-1945586ab0c5',
+            academic_cycle: ['2026'],
+            bursary_amount: '27000',
+            scholarship: '29000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:71]: update to
bursary_amount: '26000',
scholarship: '28000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 5]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'french, german, spanish 2025-2026'
+          },
+          '043c51b9-69fb-4f6e-a45c-c77329616376' => {
+            subject_name: 'german',
+            itt_categories_id: '262f596b-b8a2-40c2-9efc-1945586ab0c5',
+            academic_cycle: ['2026'],
+            bursary_amount: '27000',
+            scholarship: '29000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:79]: update to
bursary_amount: '26000',
scholarship: '28000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 6]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'french, german, spanish 2025-2026'
+          },
+          'dc383c9f-b459-4635-b83d-9adc8b1d52ba' => {
+            subject_name: 'spanish',
+            itt_categories_id: '262f596b-b8a2-40c2-9efc-1945586ab0c5',
+            academic_cycle: ['2026'],
+            bursary_amount: '27000',
+            scholarship: '29000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:87]: update to
bursary_amount: '26000',
scholarship: '28000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 7]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'french, german, spanish 2025-2026'
+          },
+          'bf92df81-2584-4e94-96a5-5b72fedb3fd6' => {
+            subject_name: 'languages',
+            itt_categories_id: '262f596b-b8a2-40c2-9efc-1945586ab0c5',
+            academic_cycle: ['2026'],
+            bursary_amount: '27000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:94]: update to
bursary_amount: '26000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 8]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'languages, geography, biology, design and technology 2025-2026'
+          },
+          '4a802b67-1cab-4185-a389-1855d909af9c' => {
+            subject_name: 'geography',
+            itt_categories_id: '518be3b3-a0b3-45cb-9299-dfaeea0c4b7e',
+            academic_cycle: ['2026'],
+            bursary_amount: '27000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:101]: update to
bursary_amount: '26000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 9]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'languages, geography, biology, design and technology 2025-2026'
+          },
+          '8919ab70-64ad-4c51-bf73-61d377e6422e' => {
+            subject_name: 'biology',
+            itt_categories_id: 'b51a9678-0a36-4361-983a-fc74f46c1b06',
+            academic_cycle: ['2026'],
+            bursary_amount: '27000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:108]: update to
bursary_amount: '26000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 10]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'languages, geography, biology, design and technology 2025-2026'
+          },
+          'aba9c49c-3b43-4f47-a903-26cd4b35e8d2' => {
+            subject_name: 'design and technology',
+            itt_categories_id: '81900c77-7f9a-4f3a-93e9-02fcbc2ff8a5',
+            academic_cycle: ['2026'],
+            bursary_amount: '25000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:115]: update to
bursary_amount: '26000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 11]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            notes: 'languages, geography, biology, design and technology 2025-2026'
+          },
+          'ff282e02-7726-4684-87d1-7a423233b41f' => {
+            subject_name: 'english',
+            itt_categories_id: '48721ee7-577b-4819-8f92-b35d48414443',
+            academic_cycle: ['2026'],
+            bursary_amount: '10000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:122]: update to
bursary_amount: '5000',

File: lib/dfe/reference_data/itt/incentives.rb [hunk 12]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            itt_categories_id: '69b468df-f7ca-40cb-9410-ed5f4e5bd61b',
+            academic_cycle: ['2026'],
+            bursary_amount: '10000',
+            notes: 'english, art & design, drama, music, re 2025-2026'
+          },
+          '3c038bba-ab48-4b14-8936-f7e2d61019fd' => {
+            subject_name: 'drama',
+            itt_categories_id: '41f02e8d-de4e-4eff-b02d-a981216ee39e',
+            academic_cycle: ['2026'],
+            bursary_amount: '10000',
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/incentives.rb:136]: Bursary removed for Drama

File: lib/dfe/reference_data/itt/subjects.rb [hunk 13]
```diff
@@ -530,6 +555,9 @@ module ITT
                                                       hecos_code: '100069',
                                                       register_category: 'e46e935b-d692-431f-bf0d-b0aad57b6035',
                                                       publish_category: '13',
+                                                      incentive: {
+                                                        '2025-2026' => 'bf92df81-2584-4e94-96a5-5b72fedb3fd6'
+                                                      },
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/subjects.rb:560]: This code links to languages and not drama

File: lib/dfe/reference_data/itt/subjects.rb [hunk 14]
```diff
@@ -743,7 +786,8 @@ module ITT
                                                       publish_category: 'W1',
                                                       incentive: {
                                                         '2020-2021' => 'd4024ca2-da1a-47ff-b497-62fed8fa5249',
-                                                        '2024-2025' => 'a324ae47-d9db-4eb5-9039-c767b8ec4f08'
+                                                        '2024-2025' => 'a324ae47-d9db-4eb5-9039-c767b8ec4f08',
+                                                        '2025-2026' => '4427e0e7-e771-4538-8224-be8d570c7996'
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/itt/subjects.rb:790]: Both 2025 and 2026 link to art and design, but shouldn't graphic design be linked to D&T? Who can we check with?

File: lib/dfe/reference_data/itt/subjects.rb (hunk 13 above)
Comment (alexchristensenedu) [reply lib/dfe/reference_data/itt/subjects.rb:560]: This code links to drama 3c038bba-ab48-4b14-8936-f7e2d61019fd - but not sure it's needed as there is no bursary any more

**Review by alexchristensenedu — CHANGES_REQUESTED** [review]
Reviewed amounts and code - changes required based on latest bursary info

File: lib/dfe/reference_data/itt/incentives.rb [hunk 15]
```diff
@@ -30,6 +30,126 @@ module ITT
       INCENTIVES = DfE::ReferenceData::HardcodedReferenceList.new(
         # SEE https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-funding-manual
         {
+          ## 2025-2026
+          'bf4ea64c-7f0d-4604-855d-b2d7a8fa3db8' => {
+            subject_name: 'mathematics',
+            itt_categories_id: '71cbc6f2-9b24-4504-86f5-979e573de058',
+            academic_cycle: ['2026'],
+            bursary_amount: '30000',
```
[the remark this answers is not in the document]
Comment (ericaporter) [reply lib/dfe/reference_data/itt/incentives.rb]: Updated to the new value

File: lib/dfe/reference_data/itt/incentives.rb [hunk 16]
```diff
@@ -30,6 +30,126 @@ module ITT
...
+            scholarship: '32000',
```
[the remark this answers is not in the document]
Comment (ericaporter) [reply lib/dfe/reference_data/itt/incentives.rb]: Updated to the new value

File: lib/dfe/reference_data/itt/incentives.rb (hunk 1 above)
Comment (ericaporter) [reply lib/dfe/reference_data/itt/incentives.rb]: Updated to the new value

File: lib/dfe/reference_data/itt/incentives.rb (hunk 3 above)
Comment (ericaporter) [reply lib/dfe/reference_data/itt/incentives.rb]: Updated to the new value

File: lib/dfe/reference_data/itt/subjects.rb (hunk 13 above)
Comment (ericaporter) [reply lib/dfe/reference_data/itt/subjects.rb]: Good spot, removed now anyway

**Review by alexchristensenedu — APPROVED** [review]
lgtm - thanks

File: lib/dfe/reference_data/itt/subjects.rb (hunk 14 above)
Comment (ericaporter) [reply lib/dfe/reference_data/itt/subjects.rb]: I've removed the link to the incentive for 2025-2026 and created --> https://trello.com/c/WEwqapWg/2312-dfe-reference-data-confirm-if-graphic-design-should-be-associated-with-a-dt-incentive-or-an-ad-incentive?filter=member:portererica

**Pull request merged**